src/mixins/filing-template-mixin.ts

```diff
@@ -126,7 +126,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
         contactPoint: this.getContactPoint,
         nameRequest: this.getNameRequest,
         offices: this.getOfficeAddresses,
-        type: this.getCorrectionErrorType
+        type: this.getCorrectionErrorType,
+        startDate: this.getCorrectionStartDate
```

Comment:
You should add this conditionally for firm corrections only. (This code already exists on lines 188-190 but please update it as needed.)

Reply:
Done!

Reply:
Please delete this line.

Also please delete the comment on line 190 (I'm assuming the logic is correct there).

Reply:
Done

Reply:
Done